docs/setup_vscode.md

```diff
@@ -310,8 +307,27 @@ Hello World!
 ```
 </div>
 
-### Create `launch.json`
-{: .primer-spec-toc-ignore }
+### macOS create `launch.json`
+
+Select the file you would like to run.  Navigate to the debugging pane.
+
+<img src="images/vscode030.png" width="768px" />
+
+Click "create a launch.json file".
+
+<img src="images/vscode031.png" width="768px" />
+
+Select LLDB.
+
+<img src="images/vscode031b.png" width="768px" />
+
+FIXME: check for incoming link to "Edit `launch.json` program"
```

Comment:
FIXME
I think we can do the cool grep thing, but I forgot the command you showed me...